packages/luna/src/components/Lessons/LessonCard.stories.tsx

```diff
@@ -123,10 +170,45 @@ export const CanceledByStudent: Story = {
     canceled: "student",
     start: new Date().toISOString(),
     duration: 30,
-    tutor: {
+    member: {
+      id: 30,
+      name: faker.person.fullName(),
+      image: url,
+      role: "student",
+    },
+    onJoin: () => console.log("join"),
+    onCancel: () => console.log("cancel"),
+  },
+};
+
+export const CanceledByCurTutor: Story = {
+  args: {
+    canceled: "tutor",
+    start: new Date().toISOString(),
+    duration: 30,
+    member: {
+      id: 30,
+      name: faker.person.fullName(),
+      image: url,
+      // NOTE: this is the role of the other user
+      role: "student",
+    },
+    onJoin: () => console.log("join"),
+    onCancel: () => console.log("cancel"),
+  },
+};
+
+export const CanceledByCurStudent: Story = {
+  args: {
+    canceled: "student",
+    start: new Date().toISOString(),
+    duration: 30,
+    member: {
       id: 30,
       name: faker.person.fullName(),
       image: url,
+      // NOTE: this is the role of the other user
```

Comment:
We can remove this note. It will be available if you hovered over the prop. 

```suggestion
```

Reply:
Any time a team member has a trouple understanding a function param or a component prop, he should check its JS Docs.